packages/examples/source/src/endpoint/example.ts

```diff
@@ -1,5 +1,5 @@
 import { Requester, Validator } from '@chainlink/ea-bootstrap'
-import { Config, ExecuteWithConfig } from '@chainlink/types'
+import { Config,  ExecuteWithConfig} from '@chainlink/types'
```

Comment:
Looks like Prettier hasn't run to format the code.

What code editor do you use?

Reply:
I use VS code

Reply:
What default formatter do you have set?

If you open the command palette and type `Format Document With` and choose `Configure Default Formatter` at the bottom, what does it say?